atom/browser/api/atom_api_window.cc

```diff
@@ -90,6 +90,15 @@ Window::Window(v8::Isolate* isolate, v8::Local<v8::Object> wrapper,
     if (options.Get("transparent", &transparent))
       web_preferences.Set("transparent", transparent);
 
+    // Offscreen windows are always created frameless.
+    bool offscreen;
+    if (web_preferences.Get("offscreen", &offscreen)) {
```

Comment:
This line could just be:

```cpp
if (web_preferences.Get("offscreen", &offscreen) && offscreen) {
```

This style is used in `atom/browser/web_contents_preferences.cc`